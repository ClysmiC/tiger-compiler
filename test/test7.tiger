let
        type int_arr := array [100] of int;
        var a : int_arr := 0;
in
        for i := 2 to 100 do
                a[i] := a[i] + a[i-1];
                printi(a[i]);
        enddo;

        break;

        a + + +
end
