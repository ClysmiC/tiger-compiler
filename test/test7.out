LET TYPE ID ASSIGN
Parser error (line 2):  type int_arr :=<--- 
                        ":=" is not a valid token. Expected "=".
unsuccessful parse
